version.go

```diff
@@ -0,0 +1,4 @@
+package main
+
+// updated with release version during docker build
+const ReleaseVersion = "RELEASE_VERSION"
```

Comment:
normally when I am doing a replace like this I do something like

```
const ReleaseVersion = "{{RELEASE_VERSION}}"
```

this helps me to remember that it is going to be replaced at build or run-time

Reply:
Very reasonable. I _thought_ I was doing something like that but must have messed it up when I moved this var into its own file. I'll update it.